syft/presenter/json/image.go

```diff
@@ -16,14 +20,23 @@ type Layer struct {
 	Size      int64  `json:"size"`
 }
 
-func NewImage(src scope.ImageSource) *Image {
+type Distro struct {
```

Comment:
Just a nit, but would architecture information be a useful include field?

Reply:
That is not available information from the current parsers that provide distro information. In most cases it is looking into `/etc/os-release` which doesn't provide that. For example, this is Debian 8:

```
PRETTY_NAME="Debian GNU/Linux 8 (jessie)"
NAME="Debian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=debian
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

Reply:
ack.